middleware.go

```diff
@@ -79,8 +79,6 @@ func parseRequestHeader(c *Client, r *Request) error {
 
 	if IsStringEmpty(hdr.Get(hdrUserAgentKey)) {
 		hdr.Set(hdrUserAgentKey, fmt.Sprintf(hdrUserAgentValue, Version))
-	} else {
-		hdr.Set("X-"+hdrUserAgentKey, fmt.Sprintf(hdrUserAgentValue, Version))
```

Comment:
what does that do? Append to the UA? If so I would also advice against this since it leaks information about the client/version (just like you shouldnt expose your webservers or os version on publicly visible status pages).